src/Core/Services/MetadataProviders/Converters/ObjectConverter.cs

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Azure.DataApiBuilder.Core.Services.MetadataProviders.Converters
+{
+    /// <summary>
+    /// This is a converter to serialize and deserialize object type
+    /// can object can be of different types int, long, string, datetime. this converter handles different basic object types supported by
```

Comment:
is this applicable to hot chocolate at all? Isn't this to deserialize default values for database columns? this should also be documented as the use case here.

Reply:
updated the comment